README.md

```diff
@@ -56,3 +56,15 @@ One recommended way can be:
     * Lists alert policies: `gcloud alpha monitoring policies list --project all-of-us-rw-prod --format json`
 3. Create/Replace the new json file.
 4. Apply this change to other envs.
+
+### Artifact Registry
+Google Artifact Registry allows us to provide Docker Hub access to members of the Registered and Controlled Tiers.
```

Comment:
this section is unrelated, but I added it for completion.  How does this look @yonghaoy ?